jazz-build-module/scm-module.groovy

```diff
@@ -166,22 +166,22 @@ def setRepoPermissions(repo_owner, repo_name, admin_group) {
     }
 }
 
-def addWebhook(repo_name, webhookName, targetUrl) {
+def addWebhook(repo_name, webhookName) {
```

Comment:
targetUrl has to be a parameter, no passing Jenkins info in here.

Reply:
In cooperated the review comment